### DISCUSSION TITLE
How to create a Dropdown that always scrolls its list to the bottom?

### DISCUSSION THREAD

**Discussion by znyi** [ant-design/ant-design#48285] [Q&A] [answered]
I am trying to create an automatically scrolled Dropdown using antd@5.16.0.

I want to have the dropdown list to always scrolled to the end on every mouse hover.

[This](https://codesandbox.io/p/sandbox/lively-breeze-ztuv4?file=%2Findex.js%3A9%2C1-36%2C25) is an example of what I want to achieve.

I found this from another stack overflow [question](https://stackoverflow.com/questions/64294551/how-to-scroll-at-the-bottom-of-the-div-onmount-using-antd-drawer), but it is using another component Drawer with an older version of antd, which is antd@4.7.0.

As u can see from [my code](https://codepen.io/znyi/pen/vYMpXJx?editors=0010), the dropdown list doesn't scroll till the end on hover. It only scrolls when the mouse leaves the Dropdown component. The list also will not scroll till the end after new item is added to the list (by clicking the Add button).

How to achieve the same auto-scrolled effect while keeping the same antd@5.16.0 version?

This question is also asked in [stack overflow](https://stackoverflow.com/questions/78277879/antd-auto-scrolled-dropdown-in-react).

**Comment by kiner-tang** [comment, marked as answer]
Maybe you can try this way to solve you problem.
https://codepen.io/kiner/pen/mdgXpeV?editors=0010
![image](https://github.com/ant-design/ant-design/assets/10286961/72af82cc-c716-40ea-baa1-d07f5302a555)
